src/services/CozoDb/cozoDb.ts

```diff
@@ -38,20 +38,32 @@ function createCozoDb() {
   const cyblogCh = createCyblogChannel({ thread: 'cozo' });
 
   const loadCozoDb = async () => {
-    db = await CozoDb.new_from_indexed_db(
-      DB_NAME,
-      DB_STORE_NAME,
-      onIndexedDbWrite
-    );
+    if (!process.env.IS_TAURI) {
```

Comment:
maybe this flag better rename to `IS_DESKTOP`?

Reply:
maybe